libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -1452,13 +1458,40 @@ async fn pull_transactions(
         .map(TryInto::try_into)
         .collect();
 
+    let outbound_transactions: NodeResult<Vec<Payment>> =
+        htlc_expiry(outbound_transactions.unwrap(), htlc_list);
```

Comment:
To avoid a potential panic
```suggestion
        htlc_expiry(outbound_transactions?, htlc_list);
```